pages/common/aws-glue.md

```diff
@@ -0,0 +1,29 @@
+# aws glue
+
+> CLI for AWS Glue.
+> Defines the public endpoint for the AWS Glue service.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/glue/index.html>.
+
+- List jobs:
+
+`aws glue list-jobs`
+
+- Start a job:
+
+`aws glue start-job-run --job-name {{job_name}}`
+
+- Start workflow run:
```

Comment:
What does this actually do?

Reply:
A workflow is a way to create dependencies between jobs using a directed acyclic graph. This ensures that a job runs only when all the other jobs its dependent on have succeeded. This command in particular will trigger the start of the workflow. Here is more info: https://docs.aws.amazon.com/glue/latest/dg/orchestrate-using-workflows.html

Reply:
Ah, I see. Thanks for the explanation! 
Perhaps this will read better?
```suggestion
- Start running a workflow:
```

Reply:
No problem! changed as suggested!